tools/curiostack-bom/build.gradle.kts

```diff
@@ -284,7 +284,7 @@ val DEPENDENCY_SETS = listOf(
         DependencySet(
                 "org.eclipse.jgit",
                 "5.10.0.202012080955-r",
-                listOf("org.eclipse.jgit")
+                listOf("org.eclipse.jgit, org.eclipse.jgit.ssh.jsch")
```

Comment:
@danada Oops - I missed this in the review too, but it's missing quotation marks :P Will send a fix

Reply:
Yikes, sorry about that